src/main/java/uk/gov/ida/verifyserviceprovider/services/NonMatchingAssertionService.java

```diff
@@ -1,12 +1,9 @@
 package uk.gov.ida.verifyserviceprovider.services;
 
 import org.opensaml.saml.common.SAMLVersion;
-import org.opensaml.saml.saml2.core.Assertion;
-import org.opensaml.saml.saml2.core.AuthnStatement;
-import org.opensaml.saml.saml2.core.StatusCode;
+import org.opensaml.saml.saml2.core.*;
```

Comment:
Can you set your IDE to not use "wildcard imports"

Setting in IntelliJ is Preferences -> Editor -> Code Style -> Java -> Imports and set the two settings to 9999

Reply:
Fixed imports👍🏼